content/templates/5-breakground/breakground.txt

```diff
@@ -0,0 +1,62 @@
+Title: Breakground
```

Comment:
We don't need to make a new template here because the Breakground team is the only ones using this. This area is really only for the front end so people can preview what options they have and most people won't have the option for the Breakground template.